docs/content/integrations/airflow.mdx

```diff
@@ -7,6 +7,9 @@ description: Explore the options for migrating from Airflow to Dagster.
 
 Migrating from Airflow to Dagster, or integrating Dagster into your existing workflow orchestration stack can be accomplished in many ways. The <a href="/guides/migrations/migrating-airflow-to-dagster">Pick your own journey</a> guide provides a variety suggestions in how one can migrate their Airflow pipelines to Dagster, or build a platform where both tools co-exist.
 
+- [Learning Dagster from Airflow](/integrations/airflow/from-airflow-to-dagster) - a step-by-step tutorial of mapping concepts from Airflow to Dagster
+- [Migrating from Airflow](/guides/migrations/migrating-airflow-to-dagster) - migration patterns for translating Airflow code to Dagster
+
 ---
 
 ## Mapping Airflow concepts to Dagster
```

Comment:
I'm inclined to say no. We could put this in the *Learning Dagster from Airflow guide*, since the guide focuses on a code-version walkthrough of this content. WDYT?

Reply:
I like that idea - I've moved the table to the bottom of the guide here:

https://github.com/dagster-io/dagster/pull/22637/commits/c9889b2801c1fbabe49b9e00c6f2c3b7a2d8dbd6

I think I'll need to revamp that guide in a follow-up PR as it still references `@op` as the best practice, we'll see!